app/policies/g5_updatable/client_policy.rb

```diff
@@ -9,9 +9,13 @@ def resolve
 
       def client_roles
         G5Authenticatable::Role
```

Comment:
why submitting this query changes if they don't really add / removes any logic? (just adding aliases). Maybe for the sake of github history it makes sense to leave them as they were